R/run_model.R

```diff
@@ -24,6 +24,11 @@ run_model <- function(data, options, path_results, language = NULL) {
       )
     )
 
+    if (!is.null(options$mock_model_trigger_error) &&
+        options$mock_model_trigger_error == "true") {
```

Comment:
if there's a chance that this option might be missing in prod you might use `identical(opptions$mock_model_trigger_error, "true")`. Any reason not to be using TRUE/FALSE here though rather than strings?

Reply:
Mention half above but the dynamic form only works with strings in SelectControl so this will be returned to as a string atm. Would save us some conversion pain if we updated that to be able to take booleans too but probably something for another ticket

Reply:
definitely another ticket!